scm_test.go

```diff
@@ -1,17 +1,31 @@
 package main
 
 import (
+	"os"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestGetScm(t *testing.T) {
 	t.Run("This project uses Git", func(t *testing.T) {
-		scm := GetScm()
+		workingDir, err := os.Getwd()
+		if err != nil {
+			workingDir = "."
+		}
```

Comment:
Do worth moving it to a utils class or something reusable?

Reply:
It's used only twice, here in the tests and in the init function of the main program, so I think we could start a new helper method for that. Let me add it, named `getDefaultwd()`

Reply:
Done in https://github.com/mdelapenya/junit2otlp/pull/22/commits/f35d631c8e19d76398137a118ea31eaf8c23d8ba